### ISSUE TITLE
ER: Update Project Profile: Expunge Assist

### ISSUE THREAD

**Issue by sylvia-nam** [hackforla/website#6579]
### Dependency
- [x] https://github.com/hackforla/website/issues/7379

### Project Name

Expunge Assist

### Project Card URL

expunge-assist.md

### Update the Title?

NO

### Update the Description?

YES

### Description

- [x] Expunge Assist is a free digital tool designed to aid Californians with arrest and conviction histories in clearing their criminal records. A personal statement is the most important proof of rehabilitation filed with an expungement petition. Our letter generator helps people write their statements to the court. The tool does not rely on AI, protects personal privacy, and does not store sensitive information. Community organizers, government agencies, and legal advocates in the clean slate and criminal justice fields are making an incredible impact. Expunge Assist aims to support that work.

### Update the Hero Image?

NO

### Update the Image?

NO

### Update the Technologies?

NO

### Update the Partner(s)?

NO

### Update the Tools?

YES

### Tools
- [x] Figma, Miro, Zoom, Google Suite, FigJam, LettuceMeet

### Update GitHub Link?

NO

### Update Slack Link?

NO

### Update Test Site Link?

NO

### Update Demo Site Link?

NO

### Update Site Link?

NO

### Remove person(s) and info from Leadership?

NO

### If YES on remove person(s), edit the below to provide the information required. If NO, delete the template in the field below:

Name:

### Add person(s) and info to Leadership?

NO

### If YES on add person(s), edit the below to provide the information required. If NO, delete the template in the field below:

Name:
GitHub Handle:
Role:
Slack Member ID:

### Project Status

Active

**Comment by sylvia-nam** [comment]
Can you also add an Overview link connected to this [page](https://docs.google.com/document/d/1Xwbyumj7zm5jluCh2MWHUs2mXwU4xXNoUxgUuWTzGaE/edit)

This is a follow-up to PM and Org meeting from a few weeks ago

**Comment by sylvia-nam** [comment]
And add this link to the Wiki page: https://github.com/hackforla/expunge-assist/wiki

**Comment by JessicaLucindaCheng** [comment]
# Issue Writing
- **Availability:** 6 hours
- **ETA:** Friday, April 12, 2024

**Comment by ExperimentsInHonesty** [comment]
@sylvia-nam I would like a few more changes to the one sheet.  
- We are still a little short of space, so it seems quite crammed.  
- There is redundant information in it
   - This text appears in the top
      >It does not rely on AI, protects personal privacy, and does not store sensitive information.
   - This text appears under Purpose
      >The tool does not rely on AI, protects personal privacy, and does not store sensitive information. 
- Once I have signed off on a new version upload a PDF of it to this repo and folder using the same naming convention https://github.com/hackforla/product-management/tree/master/project-one-sheets

**Comment by sylvia-nam** [comment]
@ExperimentsInHonesty I made corrections to the One Sheet so that there is no duplication (left the information in the header and took it out from the Purpose section), and created more spacing. Here's the [link](https://docs.google.com/document/d/1Xwbyumj7zm5jluCh2MWHUs2mXwU4xXNoUxgUuWTzGaE/edit) for your reference. 

Once you sign off, I will add to the repo you linked with appropriate name.

**Comment by sylvia-nam** [comment]
I revised the original issue so that the request to update project description matches the revisions made to the Project One Sheet. These revisions reflect Bonnie's [comments](https://github.com/hackforla/expunge-assist/issues/379#issuecomment-2067086999) during the 4/19/24 PM-Org meeting

**Comment by ExperimentsInHonesty** [comment]
met with them friday
left this message in their slack channel 
Bonnie
  7:01 PM
let us know on this issue once you add the new one sheet pdf to the product-management directory https://github.com/hackforla/website/issues/6579

**Comment by sylvia-nam** [comment]
Responded to Bonnie in Slack: I don't have permission to upload PDFs to the product-management repo. I opened a separate ticket. 
Updated One Sheet [here](https://docs.google.com/document/d/1PYlYrrgBJW2OFtQJ5SfZYLbAYPEO_Brj0dAkTJ2OnM8/edit) and in [PDF](https://drive.google.com/file/d/1KFQlBA2yBdv1rgvpjJAlCMDz1reGaQ3S/view?usp=drive_link) form.

**Comment by ExperimentsInHonesty** [comment]
- Responded to Sylvia's other ticket here: https://github.com/hackforla/website/issues/6859#issuecomment-2144025602 and here https://github.com/hackforla/website/issues/6859#issuecomment-2144026457

Waiting on resolution.  

The changes to the description and tools above are accepted.

**Comment by amejiamesinas** [comment]
EA One Sheet Submitted to PM Repo.

**Comment by ExperimentsInHonesty** [comment]
Additional changes to what is specified above are:
- [ ] Add a link to their [wiki](https://github.com/hackforla/expunge-assist/wiki) to the links in the project card 
- [ ] Upload [this pdf file](https://github.com/hackforla/product-management/blob/master/project-one-sheets/Expunge-Assist-One-Sheet.pdf) to the website repo and then add a link to the document in the project card, call the link Overview

**Comment by ExperimentsInHonesty** [comment]
The changes requested (in the top and in my prior comment, are approved.  This issue is ready for issue making.

**Comment by ExperimentsInHonesty** [comment]
Create a new issue

- [x] Use the below to make a new small issue
- [x] add the issue link you create to this ER as a dependency
- [x] move this ER to the icebox

Title of issue:
```
 Update Project Profile: Expunge Assist with new description, add tools, wiki and overview links
 ```
Body of issue
````
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [ ] In your IDE, open the `_projects/expunge-assist.md` file.
- [ ] Observe the existing syntax of the front matter block [^1] in the file.
- [ ] Add the following lines of code after `location` variable and its data
   ```
   tools: 
     - Figma
     - Miro
     - Zoom
     - Google Suite
     - FigJam
     - LettuceMeet
   ```
- [ ] Find the `description` variable and change the following:
   **From:**
   ```
   description: 'Expunge Assist aims to help people in California with criminal records accomplish record clearance, expungement or reduction and subsequently a second chance as a part of society.
   <br /><br />
   We work with verifiable non-profits, the government and partners to build digital tools that can affect changes in the lives of these justice impacted individuals.'
   ```
   **To:**
   ```
   description: 'Expunge Assist is a free digital tool designed to aid Californians with arrest and conviction histories in clearing their criminal records. A personal statement is the most important proof of rehabilitation filed with an expungement petition. Our letter generator helps people write their statements to the court. The tool does not rely on AI, protects personal privacy, and does not store sensitive information. Community organizers, government agencies, and legal advocates in the clean slate and criminal justice fields are making an incredible impact. Expunge Assist aims to support that work.'
   ```
- [ ] Download the PDF from here: https://github.com/hackforla/product-management/blob/master/project-one-sheets/Expunge-Assist-One-Sheet.pdf
- [ ] Upload the PDF to this directory: `assets/pdfs`
- [ ] Under the `links` variable, add this line of code before the other links:
   ```
     - name: Overview
        url: '../assets/pdfs/Expunge-Assist-One-Sheet.pdf'
   ```
- [ ] In the links section, after the GitHub link, add this line of code:
   ```
     - name: Wiki
       url:  'https://github.com/hackforla/expunge-assist/wiki'
   ```
- [ ] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [ ] Expunge Assist page [^2]
  - [ ] the project's card on the Projects' page [^3]  
- [ ] Once your pull request is merged, go to #6579 and
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #6579] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #6579
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/expunge-assist
[^3]: Projects page URL: https://www.hackforla.org/projects/ 
````

**Comment by HackforLABot** [comment]
Hi @dcotelessa, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by HackforLABot** [comment]
Hi @dcotelessa, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by HackforLABot** [comment]
Hi @Thinking-Panda, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by Thinking-Panda** [comment]
Availability: M-Th 10 AM- 3 PM
ETA: EOD 9/3/24

**Comment by t-will-gillis** [comment]
Hi @Thinking-Panda Issue #7379 is closed, and I have moved this issue to "Questions/ In Review" per the notes on the issue. 
- should this issue be closed?
- if not, should this issue receive a `Ready for...` label so that someone reviews it?
Thanks!

**Comment by sylvia-nam** [comment]
@t-will-gillis @Thinking-Panda - yes, it's good to close because the page was updated

**Comment by t-will-gillis** [comment]
Thanks @sylvia-nam!
